execCommand.html

```diff
@@ -795,8 +772,7 @@ <h2 id="methods-to-query-and-execute-commands">Methods to query and
                         <a class="external" data-anolis-spec="dom" href=
                         "http://dom.spec.whatwg.org/#concept-event" title=
                         "concept-event">event</a> at <var title="">affected
-                        editing host</var> that uses the <code><a href=
-                        "#editingbeforeinputevent">EditingBeforeInputEvent</a></code>
+                        editing host</var> that uses the <a>EditingBeforeInputEvent</a>
                         interface. The event's <code class="external"
                         data-anolis-spec="dom" title="dom-Event-type"><a href=
```

Comment:
heh, unrelated, but `data-anolis-spec`:) wow, this spec is ooooold school!

Reply:
OMG what did even mean??

Reply:
Anolis was the thing on which ReSpec was originally based... it's what we used in the early 2000s before ReSpec and  BikeShed :)